src/main.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  geo_location_map = {
+    "Africa"        = "southafricanorth"
+    "Asia Pacific"  = "southeastasia"
+    "Australia"     = "australiaeast"
+    "Brazil"        = "brazilsouth"
+    "Canada"        = "canadacentral"
+    "Europe"        = "westeurope"
+    "France"        = "francecentral"
+    "Germany"       = "germanywestcentral"
+    "India"         = "centralindia"
+    "Japan"         = "japaneast"
+    "Korea"         = "koreacentral"
+    "Norway"        = "norwayeast"
+    "Switzerland"   = "switzerlandnorth"
+    "UAE"           = "uaenorth"
+    "UK"            = "uksouth"
+    "United States" = "centralus"
+  }
+  location = lookup(local.geo_location_map, var.location, "")
+}
+
+resource "azurerm_resource_group" "main" {
+  name     = var.md_metadata.name_prefix
+  location = local.location
+}
+
+resource "azurerm_communication_service" "main" {
+  name                = var.md_metadata.name_prefix
+  resource_group_name = azurerm_resource_group.main.name
+  data_location       = var.location
```

Comment:
Should this be?
```suggestion
  data_location       = local.location
```

Reply:
No, because we want the data location of the Comms service to be one of these:
![image](https://user-images.githubusercontent.com/8037512/213766460-c0a33b32-c04e-4789-9b84-dff90d06c861.png)
Ref: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/communication_service

And then we use the map to convert their `data_location` to an actual Azure region for the resource group deployment.